ChangeLog.d/updated_windows_apis.txt

```diff
@@ -0,0 +1,5 @@
+Requirement changes
+   * Update Windows APIs to use BCryptGenRandom and wcslen and
```

Comment:
We should be clear that the minimum version of Windows you will then be able to run Mbed TLS on will be Vista, i.e. that we're dropping support for Windows XP.